src/services/treasury/TreasuryService.test.ts

```
@@ -0,0 +1,539 @@
+      mockPrisma.oracleRate.findFirst.mockResolvedValueOnce({
+        rateUsd: new Decimal("0.000667"),
+        timestamp: now,
+      } as any);
+
+      const result = await getEnterpriseTreasury();
+
+      expect(result.byCurrency[0].transactions.fxRateSource).toBe("current");
+      expect(result.byCurrency[0].transactions.fxRateTimestamp).toEqual(now);
+    });
+
+    it("falls back to recent FX rate when current rate unavailable", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)